components/local_date_time/local_date_time.jsx

```diff
@@ -56,6 +62,7 @@ export default class LocalDateTime extends React.PureComponent {
                 className='post__time'
                 dateTime={date.toISOString()}
                 title={title}
+                id={`post_time_${postId}`}
```

Comment:
```suggestion
                id='localDateTime'
```

`LocalDateTime` component is not only used for post but also for profile popover.  I'd suggest to have an ID like `localDateTime`, and use it in spec for center post like :
```diff
- cy.get(`#post_time_${postId}`).invoke('attr', 'title')...
+ cy.get(`#CENTER_time_${postId}`).find('#localDateTime').invoke('attr', 'title')...
```

`RHS_ROOT` for the last check on RHS root timestamp.

Reply:
ok

Reply:
@saturninoabril I think we should have a convention here for id's. They should always be different in the entire app. So something like `#CENTER_time_${postId}-localDateTime`. I am 0/5 of changing it in this PR but for further PR's we should make them unique.

Reply:
I agree to have unique ID but I'm on the fence in doing that over the entire app since it might result into unnecessary dynamic element ID and passing of props (e.g. post/user ID) to grandchild components.  The end goal of adding ID is to firmly target the element by UI Automation and I do believe that the same can be achieved even if we limit the scope of uniqueness over a section of the app.  I've created a ticket [MM-14662](https://mattermost.atlassian.net/browse/MM-14662) to come up with a proposal and present to team for approval so we have guide on how we should do it.

Reply:
>  I've created a ticket MM-14662 to come up with a proposal and present to team for approval so we have guide on how we should do it.

Sounds good 👍 

> it might result into unnecessary dynamic element ID and passing of props.

True. I don't want to complicate this all that much either certainly if there is not much benefit when compare to cost in doing it.

Honestly the only reason why i say we should use a unique identifier is because of https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id. I don't know the implications of having multiple same ID's